### PR TITLE
Update metadata

### PR DESCRIPTION
**Pull request by Sjoerd1993**
Update the metadata to include the changes so for into next release.

The time in the appdata is a placeholder, it needs a time in the present/past in order to build in the flathub beta (or stable for that matter).

### CODE REVIEW

File: data/se.sjoerd.Graphs.appdata.xml.in.in [hunk 1]
```diff
@@ -47,60 +47,80 @@
...
+          <li>The used axes limits are now saved when saving/loading a project</li>
+          <li>The preferences menu has been redesigned for a more intuitive navigation</li>
+          <li>Transparent colours for the lines are now supported</li>
+          <li>Headers are now written to exported files if present in the originally imported data</li>
+          <li>Several linguistic changes were made, to get a clearer and more consistent description</li>
+          <li>Importing column data is now more robust, supporting expressions as data points</li>
+        </ul>
+        <p>Bugfixes and changes under the hood:</p>
+        <ul>
+          <li>Translate and multiply buttons are now handles safely without calling the Python eval function</li>
```
Comment (cmkohnen) [inline data/se.sjoerd.Graphs.appdata.xml.in.in:64]:
```suggestion
          <li>Translate and multiply buttons are now handled safely without calling the Python eval function</li>
```

File: data/se.sjoerd.Graphs.appdata.xml.in.in [hunk 2]
```diff
@@ -47,60 +47,80 @@
...
+          <li>The used dependencies have been updated to the latest available releases</li>
```
Comment (cmkohnen) [inline data/se.sjoerd.Graphs.appdata.xml.in.in:65]:
```suggestion
```
I don't think this needs to be mentioned

File: data/se.sjoerd.Graphs.appdata.xml.in.in (hunk 1 above)
Comment (Sjoerd1993) [reply data/se.sjoerd.Graphs.appdata.xml.in.in:64]: Agreed, that's a better wording

File: data/whats_new [hunk 3]
```diff
@@ -1,33 +1,16 @@
...
+  <li>The preferences menu has been redesigned for a more intuitive navigation</li>
+  <li>Transparent colours for the lines are now supported</li>
+  <li>Headers are now written to exported files if present in the originally imported data</li>
+  <li>Several linguistic changes were made, to get a clearer and more consistent description</li>
+  <li>Importing column data is now more robust, supporting expressions as data points</li>
+</ul>
+<p>Bugfixes and changes under the hood:</p>
+<ul>
+  <li>Translate and multiply buttons are now handles safely without calling the Python eval function</li>
+  <li>The used dependencies have been updated to the latest available releases</li>
```
Comment (cmkohnen) [inline data/whats_new:14]:
```suggestion
```

File: data/whats_new [hunk 4]
```diff
@@ -1,33 +1,16 @@
...
+  <li>The used axes limits are now saved when saving/loading a project</li>
+  <li>The preferences menu has been redesigned for a more intuitive navigation</li>
+  <li>Transparent colours for the lines are now supported</li>
+  <li>Headers are now written to exported files if present in the originally imported data</li>
+  <li>Several linguistic changes were made, to get a clearer and more consistent description</li>
+  <li>Importing column data is now more robust, supporting expressions as data points</li>
+</ul>
+<p>Bugfixes and changes under the hood:</p>
+<ul>
+  <li>Translate and multiply buttons are now handles safely without calling the Python eval function</li>
```
Comment (cmkohnen) [inline data/whats_new:13]:
```suggestion
  <li>Number inputs are now handled safely without calling the Python eval function</li>
```

**Review by cmkohnen — APPROVED** [review]

**Pull request merged**